src/main/ruby/core/numeric_mirror.rb

```diff
@@ -47,15 +47,17 @@ def step_float_size(value, limit, step, asc)
         end
       end
 
-      def step_size(limit, step)
-        values = step_fetch_args(limit, step)
+      def step_size(limit, step, by)
```

Comment:
Are `step` and `by` the same thing?
If so, would it not be easier/cleaner to pass the right one as for `limit/to`?